client/state/data-layer/wpcom/me/two-step/application-passwords/new/test/index.js

```diff
@@ -53,3 +58,11 @@ describe( 'handleAddError()', () => {
 		} );
 	} );
 } );
+
+describe( 'apiTransformer()', () => {
+	test( 'should transform original response for a successful request', () => {
+		const appPassword = 'abcd 1234 efgh 5678';
+
+		expect( apiTransformer( { application_password: appPassword } ) ).toEqual( appPassword );
```

Comment:
can be `toBe`

Reply:
https://github.com/Automattic/wp-calypso/pull/23686/commits/9c8219d121af29abf4c60e54676fbd1d745f6073